uber/models.py

```diff
@@ -630,9 +630,22 @@ def everything(self, location=None):
                                         .order_by(Attendee.full_name).all()
                          if c.AT_THE_CON or not location or int(location) in a.assigned_depts_ints]
             for job in jobs:
-                job._available_staffers = [a for a in attendees if not job.restricted or a.trusted]
+                job._available_staffers = [a for a in attendees if not job.restricted or a.trusted_in(job.location)]
             return jobs, shifts, attendees
 
```

Comment:
So what should be done here.  If I rename this back to ._available_staff and add cached_property back to the other Job properties, would that be the original behavior?

And also, so what you're saying is that we'd call session.everything() on a page that we expect to retrieve a bunch of Jobs, which later in that page would access their cached values, and then these cached value don't persist (effectively) beyond that page load (more precisely don't persist past that session), is that correct?